### PR TITLE
Add file Hello

### PR DESCRIPTION
**Pull request by shawnlin305**
Wow! It's a goo idea... (Really ?)

### CODE REVIEW

**Comment by bcbcarl** [comment]
It looks good for me. :+1:

**Comment by taipulin** [comment]
It looks good for me. +2

**Comment by bcbcarl** [comment]
:+1:

File: Hello [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi, Git Flow. Here I am! :)
```
Comment (bcbcarl) [inline Hello:1]: Hello. This is Carl.

## Title 1

### Title 2
- aa
- bb
- cc

Comment (taipulin) [inline Hello:1]: yo

Comment (shawnlin305) [inline Hello:1]: only one line... Reject!!

Comment (asilyihong) [inline Hello:1]: O.O I'm here

**Pull request merged**